src/borg/archiver.py

```diff
@@ -3509,6 +3522,8 @@ def define_borg_mount(parser):
         subparser.set_defaults(func=self.do_delete)
         subparser.add_argument('-n', '--dry-run', dest='dry_run', action='store_true',
                                help='do not change repository')
+        subparser.add_argument('--list', dest='output_list', action='store_true',
+                               help='output verbose list of archives or print repository')
```

Comment:
guess we need to remove the "or print repository" now.

the effect is only seeing the list of archives.